src/pages/developers/tutorials.js

```diff
@@ -220,11 +222,25 @@ const TutorialsPage = ({ data, pageContext }) => {
       published: tutorial.frontmatter.published,
     }))
 
+  const externalTutorials = foreignTutorials.map((tutorial) => ({
+    to: tutorial.url,
+    title: tutorial.title,
+    description: tutorial.description,
+    author: tutorial.author,
+    tags: tutorial.tags.map((tag) => tag.toLowerCase().trim()),
+    skill: tutorial.skillLevel,
+    timeToRead: tutorial.timeToRead,
+    published: tutorial.publishDate,
```

Comment:
One step further, add `lang` here:
```suggestion
    published: new Date(tutorial.publishDate).toISOString(),
    lang: tutorial.lang ? tutorial.lang : "en",
```